src/App.jsx

```diff
@@ -14,57 +15,61 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
```

Comment:
Also you can export the selectGood function separately for better code structure.

```
const selectGood = (good) => setSelectedGood(good);
```